udata/core/metrics/__init__.py

```diff
@@ -111,10 +112,12 @@ def iso(self, value):
                 'Unsupported format: {0} ({1})'.format(value, type(value)))
 
     def trigger_update(self):
-        self.need_update.send(self)
+        if not current_app.config['TESTING'] or current_app.config.get('FORCE_METRICS'):
+            self.need_update.send(self)
 
     def notify_update(self):
-        self.updated.send(self)
+        if not current_app.config['TESTING'] or current_app.config.get('FORCE_METRICS'):
```

Comment:
I'm not sure we want to restrict that to `TESTING`, I'm thinking about local load of backups for instance but that might be another issue. @noirbizarre thoughts?